test/services/school/index.test.js

```diff
@@ -219,6 +219,38 @@ describe('school service', () => {
 
 			expect(usersSchoolUpdated.systems).to.not.be.empty;
 		});
+
+		it('should sync federalState in systems', async () => {
```

Comment:
is the difference between both tests just patch or update schools? the when is not clear to me.